kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -634,7 +634,7 @@ define([
             throw new Error('More than one batch parent found');
         }
 
-        model.setItem('exec.jobs', jobArrayToIndexedObject(jobArray));
+        model.setItem('exec.jobs.byId', jobArrayToIndexedObject(jobArray));
```

Comment:
Previously, setting `exec.jobs` would wipe out any info in other keys under `exec.jobs`, including job info, which is held at `exec.jobs.info`. This ensures that the job status updates don't affect other jobs data that might be stored.

Reply:
Yuck. Good catch.